spec.html

```diff
@@ -41552,6 +41554,8 @@ <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</
   <p><emu-xref href="#sec-regexp-pattern-flags"></emu-xref> In ECMAScript 2015, If the _pattern_ argument is a RegExp instance and the _flags_ argument is not *undefined*, a new RegExp instance is created just like _pattern_ except that _pattern_'s flags are replaced by the argument _flags_. In previous editions a *TypeError* exception was thrown when _pattern_ was a RegExp instance and _flags_ was not *undefined*.</p>
   <p><emu-xref href="#sec-properties-of-the-regexp-prototype-object"></emu-xref> In ECMAScript 2015, the RegExp prototype object is not a RegExp instance. In previous editions it was a RegExp instance whose pattern is the empty string.</p>
   <p><emu-xref href="#sec-properties-of-the-regexp-prototype-object"></emu-xref> In ECMAScript 2015, `source`, `global`, `ignoreCase`, and `multiline` are accessor properties defined on the RegExp prototype object. In previous editions they were data properties defined on RegExp instances.</p>
+  <p><emu-xref href="#sec-atomics.notify"></emu-xref> In ECMAScript 2019, `Atomics.wake` has been renamed to `Atomics.notify` to prevent confusion with `Atomics.wait`</p>
+  <p><emu-xref="#sec-asyncfromsynciteratorcontinuation"></emu-xref> &amp; <emu-xref href="#sec-asyncgeneratorresumenext"></emu-xref> in ECMAScript 2019, the number of Jobs enqueued by `await` was reduced, which could create an observable difference in resolution order between a `then()` call and an `await` expression..</p>
```

Comment:
```suggestion
  <p><emu-xref="#sec-asyncfromsynciteratorcontinuation"></emu-xref> &amp; <emu-xref href="#sec-asyncgeneratorresumenext"></emu-xref> in ECMAScript 2019, the number of Jobs enqueued by `await` was reduced, which could create an observable difference in resolution order between a `then()` call and an `await` expression.</p>
```

Reply:
LGTM for the reduced number of ticks for `await`.

Can you provide some background what does Annex E refer to? Thanks!

Reply:
@MayaLekova Annex E is where intentional breaking changes in the spec are listed, but it’s not been updated since ES2015.